### PR TITLE
Githubstudent guadalajara

### PR DESCRIPTION
**Pull request by Nayecch**
Adding my pin

### CODE REVIEW

**Comment by davidnhz** [comment]
👍

**Comment by johnyibm** [comment]
Waiting

File: _pins/githubstudent.dwm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (crichID) [inline _pins/githubstudent.dwm]: @Nayecch this file's extension should be changed to .json

File: _pins/githubstudent.dwm [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+githubHandle: githubstudent 
+latitud: 20.644801
```
Comment (crichID) [inline _pins/githubstudent.dwm]: @Nayecch please correct spelling to `latitude:`

File: _pins/githubstudent.dwm [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+githubHandle: githubstudent
```
Comment (crichID) [inline _pins/githubstudent.dwm]: @Nayecch please change `githubstudent` to the username of the individual at this pin.

File: _pins/githubstudent.nayecch [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (crichID) [inline _pins/githubstudent.nayecch]: @Nayecch the file extension must be .json

File: _pins/githubstudent.poppy [hunk 5]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (crichID) [inline _pins/githubstudent.poppy]: @Nayecch the file extension must be .json

File: _pins/githubstudent.poppy [hunk 6]
```diff
@@ -0,0 +1,4 @@
...
+githubHandle: githubstudent
+latitude: 20.659699
+logitude: -103.349609
```
Comment (crichID) [inline _pins/githubstudent.poppy]: @Nayecch please fix the spelling to be `longitude`

Comment (crichID) [inline _pins/githubstudent.poppy]: @Nayecch please add three dashes to the last line of this file `---`

File: _pint/githubstudent.lumartch [hunk 7]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (crichID) [inline _pint/githubstudent.lumartch]: @Nayecch this file name should be `_pins/USERNAME.json`

File: _pint/githubstudent.lumartch [hunk 8]
```diff
@@ -0,0 +1,4 @@
...
+githubhandle: lumartch
+latitude: 20.659699
+longitude: -103.349609
```
Comment (crichID) [inline _pint/githubstudent.lumartch]: @Nayecch please add three dashes to the end of this file `---`

File: johnyibm.json [hunk 9]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (crichID) [inline johnyibm.json]: @Nayecch this file must be in the _pins directory

**Pull request closed without merging**